### PR TITLE
EventHub : Geo DR Update

### PR DESCRIPTION
**Pull request by v-Ajnava**
## Description
EventHub : Geo DR Update - 
Authorization rule related new API for Geo DR (Alias) are added 
AlternateName property is added to ArmDisasterRecovery

RestSpec PR : https://github.com/Azure/azure-rest-api-specs/pull/2087

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj [hunk 1]
```diff
@@ -18,6 +18,10 @@
 		</PackageReleaseNotes>
 		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
 	</PropertyGroup>
+  <PropertyGroup>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```
Comment (shahabhijeet) [inline src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj]: @v-Ajnava all comments apply to this PR as well that I have provided in #3948 
for both the projects

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj (hunk 1 above)
Comment (v-Ajnava) [reply src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj]: Resolved

Comment (shahabhijeet) [reply src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj]: I still see it? line 22

File: src/SDKs/EventHub/EventHub.Tests/EventHub.Tests.csproj [hunk 2]
```diff
@@ -10,6 +10,10 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```
Comment (shahabhijeet) [inline src/SDKs/EventHub/EventHub.Tests/EventHub.Tests.csproj]: The property is still being set.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/EventHub/EventHub.Tests/EventHub.Tests.csproj (hunk 2 above)
Comment (v-Ajnava) [reply src/SDKs/EventHub/EventHub.Tests/EventHub.Tests.csproj]: Resolved. Removed the property

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**